src/main/java/org/spdx/library/model/ExternalDocumentRef.java

```diff
@@ -224,6 +224,19 @@ public ExternalDocumentRef setSpdxDocumentNamespace(String documentNamespace) th
 						public String getIndividualURI() {
 							return documentNamespace;
 						}
+						
+						@Override
+						public boolean equals(Object compare) {
+							if (!(compare instanceof IndividualUriValue)) {
+								return false;
+							}
+							return ((IndividualUriValue)compare).getIndividualURI().equals(documentNamespace);
+						}
+						
+						@Override
+						public int hashCode() {
+							return documentNamespace.hashCode();
+						}
```

Comment:
Good catch - I believe this is was an incorrect addition.  I'll update the PR removing this check.

Reply:
In looking back at the code, the override is for an `IndividualUriValue` created inline for a property value, so I believe the code is correct:

```
			setPropertyValue(SpdxConstants.PROP_EXTERNAL_SPDX_DOCUMENT,
					new IndividualUriValue() {
						@Override ...
```

Reply:
Oh, you are totally right, I didn't look close enough :flushed:
But then I wonder why `SimpleUriValue` with `documentNamespace` as uri isn't used here? I assumed that this was meant as a generic implementing class of `IndividualUriValue`, and I believe it would behave the same (apart from a technical difference in `hashCode` that shouldn't matter)